### ISSUE TITLE
healthcheck script errors on MathJax

### ISSUE THREAD

**Issue by Lennart-Kramer** [ilifau/assStackQuestion#96]
Hello again,
when trying to run the healthscript, one gets an error instead and the following backtrace is generated:

```
[ea518] [2022-08-24 16:34:47.675060] default_root.ERROR: ilErrorHandling::{closure}:50 0 Class 'stack_maths_output_filter_base' not found in /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/stack/mathsoutput/mathsoutputmathjax.class.php:29#0 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/stack/mathsoutput/mathsoutput.class.php(137): include_once()
#1 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/stack/mathsoutput/mathsoutput.class.php(116): stack_maths::class_for_type('mathjax')
#2 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/stack/mathsoutput/mathsoutput.class.php(107): stack_maths::get_output_instance('mathjax')
#3 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/stack/mathsoutput/mathsoutput.class.php(44): stack_maths::get_output()
#4 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/locallib.php(71): stack_maths::process_lang_string('Check LaTeX is ...')
#5 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/model/configuration/class.assStackQuestionHealthcheck.php(75): stack_string('healthchecklate...')
#6 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/model/configuration/class.assStackQuestionHealthcheck.php(56): assStackQuestionHealthcheck->checkMaximaConnection()
#7 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/class.ilassStackQuestionConfigGUI.php(376): assStackQuestionHealthcheck->doHealthcheck()
#8 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/class.ilassStackQuestionConfigGUI.php(401): ilassStackQuestionConfigGUI->showHealthcheck(true)
#9 /var/www/html/Customizing/global/plugins/Modules/TestQuestionPool/Questions/assStackQuestion/classes/class.ilassStackQuestionConfigGUI.php(80): ilassStackQuestionConfigGUI->runHealthcheck()
#10 /var/www/html/Services/Component/classes/class.ilPluginConfigGUI.php(81): ilassStackQuestionConfigGUI->performCommand('runHealthcheck')
#11 /var/www/html/Services/UICore/classes/class.ilCtrl.php(217): ilPluginConfigGUI->executeCommand()
#12 /var/www/html/Services/Component/classes/class.ilObjComponentSettingsGUI.php(113): ilCtrl->forwardCommand(Object(ilassStackQuestionConfigGUI))
#13 /var/www/html/Services/UICore/classes/class.ilCtrl.php(217): ilObjComponentSettingsGUI->executeCommand()
#14 /var/www/html/Services/Administration/classes/class.ilAdministrationGUI.php(246): ilCtrl->forwardCommand(Object(ilObjComponentSettingsGUI))
#15 /var/www/html/Services/UICore/classes/class.ilCtrl.php(217): ilAdministrationGUI->executeCommand()
#16 /var/www/html/Services/UICore/classes/class.ilCtrl.php(178): ilCtrl->forwardCommand(Object(ilAdministrationGUI))
#17 /var/www/html/ilias.php(23): ilCtrl->callBaseClass()
#18 {main}  
```
MathJax is enabled and working in ilias, the used stack version is [6.8.3](https://github.com/ilifau/assStackQuestion/commit/fb97df6cb6e4af2fedb98d9819ad1926d07ef39a).

**Comment by jcopado** [comment]
Hello Lennart, I have released a new bugfix Version 6.8.4
I hope this time it works, I wonder why this error on freshly new installation but not in already in use one.
Let me know if it worked

Thanks

**Comment by Lennart-Kramer** [comment]
Yes it works now, thank you again!
